app/models/user.php

```diff
@@ -82,5 +83,135 @@ public function login(LoginInfo $login_info)
             throw new RecordNotFoundException('No Record Found');
         }
         Session::set('username', $this->username);
+        Session::set('id', $row['id']);
```

Comment:
setting session value is not a model's function. 
you can move this in the controller.
just don't put any logic or process in the model that is not related to database :bow: